### PR TITLE
Distribute binaries separately

### PR DESCRIPTION
**Pull request by guidanoli**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -55,3 +55,17 @@ jobs:
...
           cache-to: type=gha,mode=max,scope=ubuntu
+
+      - name: Extract binaries from image
+        if: startsWith(github.ref, 'refs/tags/v')
+        uses: shrink/actions-docker-extract@v3
+        with:
+          image: ${{ steps.build_image.outputs.imageid }}
+          path: /opt/bundle/.
+          destination: out
+
```
Comment (endersonmaia) [inline .github/workflows/build.yml:66]: Maybe a checksum step to generate a `.sha256sum` file for each binary?

It would be nice, but not a requirement.

**Review by endersonmaia — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (guidanoli) [reply .github/workflows/build.yml:66]: How about a single `SHA256SUMS.txt` file?

Comment (endersonmaia) [reply .github/workflows/build.yml:66]: It also works, don't need `.txt` extensio.

But I like individual sha256sum file per binary, so you can download like

```shell
curl https://example.com/file
curl https://example.com/file.sha256
```

But it's up to you.

Comment (guidanoli) [reply .github/workflows/build.yml:66]: Done!

**Pull request merged**